tests/MongoDB.Driver.Tests.Dotnet/Specifications/transactions/tests/abort.json

```diff
@@ -1,4 +1,6 @@
 {
```

Comment:
Updating the json files in the Dotnet test project isn't strictly related to AutoStartTransaction.

They got inadvertently out of sync because last time I updated the json files I forget that they have to be updated in two places.